app/templates/events/event_list.html

```diff
@@ -96,12 +96,15 @@ <h1 class="text-center">Events List for {{selectedTerm.description}}</h1>
               {% set link_class = 'link-secondary' if event.isPast else 'link-primary' %}
                 <td><a href="/event/{{event.id}}/edit" class="{{link_class}}">{{event.name}}</a></td>
             {% else %}
-              {% set link_class = 'link-secondary' if event.isPast else 'link-primary' %}
-              <td><a href="/event/{{event.id}}/view" class="{{link_class}}">{{event.name}}</a></td>
+              <td><a href="/event/{{event.id}}/view" class="link-primary">{{event.name}}</a></td>
             {% endif %}
-            <td nowrap>{{event.startDate.strftime('%m/%d/%Y')}}</td>
-            <td nowrap>{{event.timeStart.strftime('%-I:%M %p')}} - {{event.timeEnd.strftime('%-I:%M %p')}}</td>
-            <td class="text-dark">{{event.location}}</td>
+              {% if not user.isAdmin and event.isCanceled %}
+                <td colspan = 3 >This event has been canceled. Contact CELTS if you have any questions.</td>
```

Comment:
We should also show this message, or a similar one to admins. Also, we should say "Contact {{programManager}}..." if the program has a manager. If there is not program manager then it is fine to keep it as CELTS. We should also turn the program managers name into a mailTo link.

Reply:
Fixed